src/build/docker/arch/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM archlinux/base
+
+RUN pacman --noconfirm -Syu
+
+RUN pacman --noconfirm -S nodejs
+RUN pacman --noconfirm -S npm
+RUN pacman --noconfirm -S gcc
+RUN pacman --noconfirm -S coreutils
+RUN pacman --noconfirm -S grep
+RUN pacman --noconfirm -S tar
+RUN pacman --noconfirm -S sed
+RUN pacman --noconfirm -S gawk
+RUN pacman --noconfirm -S make
+RUN pacman --noconfirm -S diffutils
```

Comment:
Is there a reason that you write every dependency as a separate line vs `pacman --noconfirm -S nodejs npm gcc ...`?

Reply:
@tcoopman - good catch! No reason to have separate lines - your suggestion is cleaner 👍